app/components/preprint-form-project-select.js

```diff
@@ -68,13 +71,19 @@ import { task, timeout } from 'ember-concurrency';
  */
 export default Component.extend(Analytics, {
     panelActions: service('panelActions'),
-    userNodes: A(),
     selectedNode: null,
     currentPage: 1,
     searchTerm: '',
     canLoadMore: false,
     isLoading: false,
     currentPanelName: null,
+    /**
+     * Whether to show the file selection dropdown box
+     * @property {boolean} fileSelect
```

Comment:
This looks like some JSDoc stuff that belongs to top of the file or start of the class/component. Could be just a simple single line comment instead of a documentation comment.